MyStore/Settings.cpp

```diff
@@ -0,0 +1 @@
+#include "Settings.h"

```

Comment:
Settings.h and Settings.cpp files are irrelevant to this change, let's remove them from the change.